### ISSUE TITLE
SQL tab: No more horizontal stacking of query and results pane

### ISSUE THREAD

**Issue by pamtbaau** [sqlitebrowser/sqlitebrowser#3028]
Details for the issue
--------------------

#### What did you do?
Enabling horizontal tiling in sql tab preferences

#### What did you expect to see?
That result pane and query pane are horizontally stacked

#### What did you see instead?
Result pane and query pane remain vertically stacked

Useful extra information
-------------------------
> DB4S v3.12.2 [built for x86_64-little_endian-llp64] on Windows 10 (10.0) (winnt/10.0.22000) [x86_64]
> using SQLite Version 3.35.5
> and Qt 5.12.8

**Comment by chrisjlocke** [comment]
Can you provide a screenshot?
For me, each pane is 'stacked', but unsure if thats what you mean.

![image](https://user-images.githubusercontent.com/7872218/165150484-e6de6a24-d4b7-46a2-8c55-374a13844102.png)

I'd call this vertical, not horizontal.

In horizontal mode, the results are still below the results, but the results is to the right of the SQL

![image](https://user-images.githubusercontent.com/7872218/165150765-65a1d402-516b-4034-b579-dd8f84ed9681.png)

**Comment by pamtbaau** [comment]
The last image is what I mean. This is how it used to be when in `Edit/Preferences/SQL/Horizontal tiling` was set to true.

With fresh installation on Windows 11, the stacking remains vertical.

**Comment by chrisjlocke** [comment]
So click 'Edit', 'Preferences', go to the 'SQL' tab and turn on 'horizontal tiling'.

![image](https://user-images.githubusercontent.com/7872218/165153980-5fa71472-6796-4c3d-a6ba-b4a7bfae14fe.png)

You shouldn't need to restart DB4S.
Sorry - thought you'd already tried this, and you were reporting that it wasn't working...

**Comment by pamtbaau** [comment]
As said, that's what I already did, but doesn't make a difference...

**Comment by chrisjlocke** [comment]
Which version of Windows 11?  Can you do a Winver.  Also on your opening post, it states Windows 10 (which is what I'm using) and I don't see any issue on that.

**Comment by pamtbaau** [comment]
Edition	Windows 11 Home
Version	21H2
OS build	22000.613

**Comment by sky5walk** [comment]
If you observe the User SQL Log, it replaces the 3rd results window you are showing.
Just collapse that redundant box with the horizontal slider and try this:
![image](https://user-images.githubusercontent.com/13093122/165537062-b4472aea-ea3e-4ddc-933a-c0c7ab5107f4.png)

**Comment by pamtbaau** [comment]
Not sure if I understand what you are trying to get across...

**Comment by sky5walk** [comment]
Just a compromise to display data below the SQL query and the status/error on the lower left SQL log. As the picture shows.

**Comment by chrisjlocke** [comment]
> Not sure if I understand what you are trying to get across

You can reduce the size of the log pane to nothing, then add the SQL log dock and position it with more control.

**Comment by pamtbaau** [comment]
Still on Windows 11.
Just downloaded version Version 3.11.99 (Jan  1 2019). This is how it looks like using Horizontal tiling enabled.

![image](https://user-images.githubusercontent.com/5120011/165594374-0bf96a57-c921-4245-b0ee-4cb830cb6dc5.png)

Downloaded version of 2020 and 2021 which do not show horizontal tiling.

**Comment by chrisjlocke** [comment]
Can you provide a screenshot of what you do see?
I can't replicate this, and would just be interesting to see what you see.

**Comment by pamtbaau** [comment]
That is a screenshot of what I see...

**Comment by chrisjlocke** [comment]
Yes, indeed.  I was after a screenshot of what you're seeing in v3.12 which isn't working.  ie, what you are seeing, ie, "and would just be interesting to see what you see"

**Comment by pamtbaau** [comment]
Reinstalled version 3.12.2 and this is what I see:

![image](https://user-images.githubusercontent.com/5120011/165600336-6f4ea126-b9c0-42a5-b6aa-3aee66b16c06.png)

**Comment by chrisjlocke** [comment]
Just as a data point, what happens if you install a nightly version?  
https://nightlies.sqlitebrowser.org/latest/DB.Browser.for.SQLite-win32.zip

You can extract this zip and run the .exe - you don't need to install it.

I did jhave a virtual machine with Win 11 running and I'm sure I ran DB4S in it, but it was some time ago.  I'll try and reinstall it.
Thanks for your patience in providing the screenshots.

**Comment by sky5walk** [comment]
Did you try enabling the SQL log?
You do not have the SQL log displayed.

**Comment by pamtbaau** [comment]
@chrisjlocke, Installed your nightly (3.12.99 (Mar 25 2022). Same result.

@sky5walk, No I do not have SQL log enabled. I did open the SQL log window, but do not see the use of it. Main "Execute SQL" window remains displaying a vertical layout.

**Comment by sky5walk** [comment]
The SQL Log shows you your SQL errors and the app's.
It allows you to expand the available area for SQL results by collapsing the SQL status section under the Execute SQL panel.
Again, this is a compromise that works now. 💡

**Comment by pamtbaau** [comment]
Ah ok, now I understand the purpose... I think I rather keep using version 3.11.99 (Jan 1 2019) instead ;-)

**Comment by chrisjlocke** [comment]
Depending on your use of DB4S, there are many improvements from 3.11.99 to 3.12.2 - even from 3.12.2 to the nightly.  Personally, I'd just get used to the layout in 3.12.2 - it's better than using a really old version of DB4S.

**Comment by pamtbaau** [comment]
See this issue from 2015, my feature request about redesigning the SQL pane: [Split "Execute SQL" tab vertically in query & result/feedback](https://github.com/sqlitebrowser/sqlitebrowser/issues/380#). This request led to the layout shown [above](https://github.com/sqlitebrowser/sqlitebrowser/issues/3028#issuecomment-1111344534).

**Comment by chrisjlocke** [comment]
As this works in Win 10 but not in your Win 11, I'm guessing its a quirk of the OS - for some reason a value DB4S sees or a value its getting from the registry.  
And I know you'll punch me in the face for asking, but just want to check.  in Win 11, as its not working, if you click 'Edit' and 'Preferences', the 'Horizontal tiling' option is enabled?  

![image](https://user-images.githubusercontent.com/7872218/165634269-7e6a82c3-9733-4ecf-ab76-f62944575f0e.png)

If it is, can you click it so its off, then click it again?  (and click Save).  No improvement?

**Comment by pamtbaau** [comment]
I've tried switching on/off before and again per your request to be sure. It doesn't make a difference.

I also wouldn't expect it to make it a difference because version 3.11.99 and version 3.12.2 use the same configuration. Hence it cannot be a Windows 11 quirk. Also, The config of Windows 11 remains the same and only DB4S is changing.
